tsconfig.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Does sewing-kit-next expect the `tsconfig.json` to live in the root dir?

Reply:
Not certain, but I [believe so](https://github.com/lemonmade/sewing-kit/blob/main/packages/plugin-typescript/src/plugin-typescript.ts#L324)? I remember getting some errors about not finding a `quilt/tsconfig.json` when playing around with with different `tsconfig.json` setups.